plugins/woocommerce/src/Internal/DataStores/Orders/OrdersTableDataStore.php

```diff
@@ -3076,6 +3153,7 @@ protected function after_meta_change( &$order, $meta ) {
 		} else {
 			$order_cache = wc_get_container()->get( OrderCache::class );
 			$order_cache->remove( $order->get_id() );
+			$this->invalidate_cache_for_objects( array( $order->get_id() ) );
```

Comment:
I would like to leave this out as all the callers to `::after_meta_change()` already flush meta cache and there isn't really a direct reason to flush the cache for the object too.  However, having this here is needed for Woo Subscriptions compatibility when `$subscription->update_dates()` is used to update the created date. This may be a trivial incompatibility, though.

Removing this line takes off another 10 queries from the checkout flow.